src/components/BaseLink.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function BaseLink({ to, children, className }) {
+  return (
+    <Link to={to} className={`text-blue-500 hover:text-blue-700 ${className}`}>
```

Comment:
Suggestion - add a transition for the hover effect. You can leverage tailwind classes like `transition-colors` and `duration-200`

Reply:
Done!